com.unity.sg2/Documentation~/UV-Flow-Map-Node.md

```diff
@@ -21,10 +21,11 @@ The UV Flow Map Node is under the **UV** category in the Create Node menu.
 [!include[nodes-inputs](./snippets/nodes-inputs.md)] <!-- MULTIPLE INPUT PORTS INCLUDE -->
 | **Name** | **Type** | **Description** |
 | :------  | :------- | :-------------  |
-|  **Flow Map**  | Vector 2 | x and y values that define a direction in which the flow should occur.  This data is generally stored in a flow map texture and should be expanded to the -1 to 1 range before passing it into the UV Flow Map node’s Flow Map input port. |
+|  **Flow Map**  | Vector 2 | x and y values that define a direction in which the flow should occur.  This data is generally stored in a flow map texture and should be expanded to the -1 to 1 range before passing it into the UV Flow Map node's Flow Map input port. |
 |  **Strength**  | Float | a multiplier for the flow map data that controls how far the UVs are pushed in the flow direction. |
 |  **Flow Time**  | Float | the timing phase for the flow effect.  By default, this uses an internal Flow Map Time node to generate a spatially-varying phase offset for time.  For a more efficient effect, you could simply connect a time node to a frac node and pass that into Flow Time. Using the Flow Map Time node breaks up the uniform pulsing artifacts that can otherwise occur without it. |
```

Comment:
```suggestion
|  **Flow Time**  | Float | The timing phase for the flow effect.  By default, this uses an internal Flow Map Time node to generate a spatially-varying phase offset for time.  For a more efficient effect, you could simply connect a time node to a frac node and pass that into Flow Time. Using the Flow Map Time node breaks up the uniform pulsing artifacts that can otherwise occur without it. |
```